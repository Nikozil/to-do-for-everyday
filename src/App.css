.app-wrapper {
  display: grid;
  margin: 0 auto;
  height: 100vh;

  grid-template-areas:
    's h'
    's c';

  grid-template-rows: 30px 1fr;
  grid-template-columns: 60px 1fr;
}
.app-wrapper-content {
  grid-area: c;
}
@media screen and (max-width: 767px) {
  .app-wrapper {
    grid-template-areas:
      'h'
      's'
      'c';
    grid-template-rows: 25px 40px 1fr;
    grid-template-columns: 1fr;
  }
}
