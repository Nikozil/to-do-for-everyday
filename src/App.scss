$primary-color: #354052;
$secondary-color: #445269;
$header-color: #fff;
$button-color: #6c757d;
$button-color-hover: #5c636a;
$button-color-selected: #a0aeba;
$content-color: #f8f9fa;
$comment-color: #6c757d;
$A-color: #97b6e8;
$B-color: #9fc0f5;
$C-color: #86a2cf;
$text-color: #000000;
$nav-icon-color: #fff;
// $background-color: #3540522c;
$background-color: #35405210;

@mixin focus {
  border: none;
  outline: none;
  &:focus-visible {
    box-shadow: 0 0 0 0.25rem rgba(130, 138, 145, 0.5);
  }
}
@mixin focus-important {
  &:focus-visible {
    border-color: rgba(130, 138, 145, 0.5) !important;
    box-shadow: 0 0 0 0.25rem rgba(130, 138, 145, 0.5) !important;
  }
}
