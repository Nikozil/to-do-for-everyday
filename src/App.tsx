import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import { ProvideAuth } from './api/useAuth';
import './App.css';
import AppPage from './components/AppPage/AppPage';
import LoginPage from './components/Pages/LoginPage/LoginPage';
import store from './Redux/store';

const App: React.FC = () => {
  return (
    <ProvideAuth>
      <Provider store={store}>
        <BrowserRouter>
          <Switch>
            <Route path="/login">
              <LoginPage />
            </Route>
            <Route path="/">
              <AppPage />
            </Route>
          </Switch>
        </BrowserRouter>
      </Provider>
    </ProvideAuth>
  );
};

export default App;
