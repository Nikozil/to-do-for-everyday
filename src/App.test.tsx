import React from 'react';
import { render, waitFor } from '@testing-library/react';
import App from './App';
describe('App', () => {
  it('render correct', async () => {
    const { asFragment } = render(<App />);
    await waitFor(() => {
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
