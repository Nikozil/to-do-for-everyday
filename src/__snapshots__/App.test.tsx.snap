// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render correct 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <div
      class="content"
    >
      <div
        class="logo"
      >
        <img
          alt="Logo"
          src="Logo.png"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
